chpl/chpl-service/src/main/resources/errors.properties

```diff
@@ -273,6 +279,8 @@ user.notFound=User not found.
 
 #Warnings
 transparencyAttestation.save=Warning: The transparency attestation for the developer is different in the system than in the upload file. To save the updated transparency attestation, as well as all other developer information, please click 'Save as Developer Information' above before proceeding.
+listing.chplCertificationId=CHPL certification ID was not found.
+listing.acbId=ACB ID was not found.
```

Comment:
Are either of these errors referenced in the code above? They don't show up in the diff